Dockerfile

```diff
@@ -12,26 +12,25 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM golang:1.7
-MAINTAINER Quentin Machu <quentin.machu@coreos.com>
+FROM golang:1.8-alpine
+MAINTAINER Jimmy Zelinskie <jimmy.zelinskie@coreos.com>
 
 VOLUME /config
 EXPOSE 6060 6061
 
-RUN apt-get update && \
-    apt-get install -y bzr rpm xz-utils supervisor && \
-    apt-get autoremove -y && \
-    apt-get clean && \
-    rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* # 06SEP2016
-
-RUN go get -u github.com/cloudflare/cfssl/cmd/cfssl
-RUN go get -u github.com/cloudflare/cfssl/cmd/cfssljson
-RUN go get -u github.com/coreos/jwtproxy/cmd/jwtproxy # 06SEP2016
-RUN go get -u github.com/coreos/clair/cmd/clair
-RUN cd /go/src/github.com/coreos/clair && git checkout v1.2.6 && go install github.com/coreos/clair/cmd/clair
+RUN apk add --no-cache git bzr rpm xz supervisor gcc musl-dev && \
+    go get -u github.com/cloudflare/cfssl/cmd/cfssl && \
+    go get -u github.com/cloudflare/cfssl/cmd/cfssljson && \
+    apk del gcc && \
+    go get -u github.com/coreos/jwtproxy/cmd/jwtproxy && \
+    go get -u github.com/coreos/clair/cmd/clair && \
```

Comment:
you can -d if you run `go install` afterwards, so we don't compile two versions.